tests/affiliatedcertification/affiliatedcerthelper/operator.go

```diff
@@ -205,7 +205,7 @@ func IsCatalogSourceEnabled(name, namespace, displayName string) bool {
 func DeleteCatalogSource(name, namespace, displayName string) error {
 	source, err := globalhelper.APIClient.CatalogSources(namespace).Get(context.TODO(), name, metav1.GetOptions{})
 	if err != nil {
-		return err
+		return nil
```

Comment:
@theemj  I saw your comment on this commit, but something with this line is not clear to me, maybe you meant to return the err?

Reply:
I understood that if no catalogsource was found, then the err wouldn't be nil, thus I think it's better to return bool,error, incase no catalogsource was found, then return false,nil.